tests/installation/installation_overview.pm

```diff
@@ -16,12 +16,24 @@ use warnings;
 use base "y2logsstep";
 use testapi;
 
+sub sle15_workaround_broken_patterns {
+    if (check_var('VERSION', '15')) {    # SLE 15 has pattern errors, workaround them - rbrown 04/07/2017
+        while (check_screen('sle-15-failed-to-select-pattern')) {
+            record_soft_failure 'bsc#1047327';
+            send_key 'alt-o';
+        }
```

Comment:
Actually, even with check_screen while loop against multiple tags we may save time, as if sle-15-failed-to-select-pattern appeared 1 time in one run, in case of array we will match next error during same iteration, instead of failing to match previous error screen and waiting default timeout.